src/test/java/gov/healthit/chpl/aqa/pageObjects/SearchPage.java

```diff
@@ -12,9 +12,36 @@
     private static WebElement element = null;
 
     /**
-     * Returns the first search details button.
+     * Returns the acbFiltersLink.
      *
      * @param driver webdriver
+     * @return acbFilters Link
+     */
+    public static WebElement acbFiltersLink(final WebDriver driver) {
+        element = driver.findElement(By.xpath("//*[@id=\"filters\"]/div[2]/div[5]/ul/li[5]/st-list-multiple/ul/li[6]/label"));
+        return element;
+    }
+    /**
+     * Returns SLI Compliance ACB filter option.
+     * @param driver WebDriver
+     * @return SLI Compliance ACB filter option element
+     */
+    public static WebElement acbSLIFilter(final WebDriver driver) {
```

Comment:
Didn't I add an ID to this link? Shouldn't the xpath be using that?

Also, if you want this to be an awesome element, it'd make sense to pass in the ACB name and use that, in conjunction with the IDs, to allow this PageObject call to reference any of the ACB filters

Reply:
I corrected the SLI element locator to use id, i had added xpath before id was added and forgot to update. I'll work on using ACB name in conjunction with the IDs to improve this test in next update.